atomium/templates/block/block.component.inc

```diff
@@ -20,31 +20,12 @@ function atomium_preprocess_block(array &$variables, $hook) {
     $variables['classes_array'] = \array_merge($variables['classes_array'], (array) $variables['block']->classes);
   }
 
-  if (isset($variables['block']->atomium)) {
-    if (isset($variables['block']->atomium['total_blocks_in_regions'])) {
-      if (isset($variables['block']->atomium['block_index_in_region'])) {
-        $total = $variables['block']->atomium['total_blocks_in_regions'];
-        $index = $variables['block']->atomium['block_index_in_region'];
-
-        if ($index === 0) {
-          $variables['atomium']['attributes']['wrapper']->append('class', 'first');
-        }
-
-        if ($total === $index) {
-          $variables['atomium']['attributes']['wrapper']->append('class', 'last');
-        }
-
-        $variables['atomium']['attributes']['wrapper']->append('class', 'weight-' . $index);
-      }
-    }
-  }
-
   if (
+    $variables['elements']['#block']->module === 'menu_block' ||
     ($variables['elements']['#block']->module === 'system' && $variables['elements']['#block']->delta === 'navigation') ||
     ($variables['elements']['#block']->module === 'system' && $variables['elements']['#block']->delta === 'main-menu') ||
     ($variables['elements']['#block']->module === 'system' && $variables['elements']['#block']->delta === 'user-menu') ||
-    ($variables['elements']['#block']->module === 'admin' && $variables['elements']['#block']->delta === 'menu') ||
-    $variables['elements']['#block']->module === 'menu_block'
+    ($variables['elements']['#block']->module === 'admin' && $variables['elements']['#block']->delta === 'menu')
   ) {
     $variables['theme_hook_suggestions'][] = 'block__menu';
   }
```

Comment:
I would suggest the follow steps:
`
$module = $variables['elements']['#block']->module;
$delta = $variables['elements']['#block']->delta;
`

Reply:
Thanks for the commit you did, it's now fixed.